### ISSUE TITLE
Кредитная история 

### ISSUE THREAD

**Issue by Davibiyuu** [Davibiyuu/zashitacoil#356]
Всем привет. Подскажите, на что влияет кредитная история в целом?

**Comment by diorjio** [comment]
[Кредитная история](https://www.t24.su/na-chto-vliyaet-kreditnaya-istoriya/) на самом деле играет очень важную роль во многих аспектах жизни, особенно когда речь идет о получении кредитов или ипотек. Она показывает кредиторам, насколько ты ответственно подходишь к выплатам, и на основании этого они принимают решение, выдавать ли тебе кредит и на каких условиях. Советую почитать статью, там очень подробно описано, как именно кредитная история влияет на твои финансовые возможности и на что стоит обратить внимание

**Comment by CatrinStaff** [comment]
Привет, друг! Кредитная история влияет на шанс одобрения кредита, так как там хранятся все ваши кредиты, карты и многое другое. Я недавно решил попробовать заказать дебетовую карту мир [https://www.otpbank.ru/retail/cards/debit/mir/](https://www.otpbank.ru/retail/cards/debit/mir/) . Лучшим вариантом для был ОТП банк, так как меня устраивал кэшбэк и повышенная ставка по накопительному счёту. Оформил прямо через сайт!

**Comment by CatrinStaff** [comment]
Я согласен! Кредитная история очень важна в некоторых моментах, но я стараюсь позаботиться  деньгах заранее и вот недавно открыл вклад на 2 месяца [https://rencredit.ru/deposits/spec/vklad-na-2-mesyatsa/](https://rencredit.ru/deposits/spec/vklad-na-2-mesyatsa/) . Очень приятно получать доходность на свои сбережения особенно тогда, когда ставки на такие вклады очень высокие и  хорошей доходностью. Считаю, что это отличное вложение средств, а банк Ренессанс надёжный!
